xeno/shields.md

```diff
@@ -0,0 +1,112 @@
+
+4. Repeat the same buff abilities over and over on different xenos. When a xeno needs a shield-like buff ability, it should be unique to some extent. 
+
+5. Make said buff abilities overpowered and frustrating to fight.
+
+## Content
+
+Shields to be replaced in most areas by buff abilities.
+
+Buff abilities to be temporary, somewhat increase durability or survivability, or give a resistance to a xeno. They are not to be heals. 
+
+Varied types of buff ability to be made (clarity, speed boosts, etc) and given to different castes where they best fit.
+
+Buff abilities to be limited to praetorians, carriers (shaman), the queen, drone (gardener), and potentially other castes that already have buff abilities. Shaman carrier and Warden prae **must** recieve new buff abilities or offensive abilities to compensate for their loss of shields.
+
+Castes that have buff abilities that only buff themselves are not to recieve buff abilities that can buff other xenos.
+
+Shields, or buffs like them, exist to give xenos more durability and strength - basically to increase their survivability not in the form of a simple heal. It should be clear to players when this xeno is more durable, and should also not persist off the frontline or majorly interfere with how you fight a specific caste when they are applied. 
+
+A "buff ability" is a replacement for shields, and is the name that is used to refer to any ability that can replace shields.
+
+Clarity stacks are limited effect stacks that let xenos resist minor stuns, can be seen on boxer warrior. These can be used as replacement buffs for shields
+
+Warden is a xeno that can provide, as one of its abilities, short-term buffs or heals. (not long term spam shields)
+
+Shaman is a xeno that can provide, as one of its abilities, short-term AOE buffs or AOE heals. (not long term shields for the entire hive such as in hijack)
+
+Warrior is a xeno that is hard to kill in a 1v1 and takes a frontliner / ambusher role with a preferably higher focus on frontlining. It is not a xeno that can easily stay far away from the main frontline without nearby weeds or assistance like lurker, it is not a xeno that can very easily disengage and run away like lurker. 
+
+## Specific changes (preliminary ideas)
+
+Shields to be removed from all but crusher, rav
+
+Warden shield - replace with a speed boost / warrior clarity stacks / fire extinguish? -> a buff that helps xenos at the front line, and can help them while they are at full health
+
+Carrier shield - replace with aoe clarity stacks / a 10s aoe clarity effect / slash damage -> some other kind of buff that makes xenos better offensively
+
+Vanguard can prob keep shield since it’s just a one free hit mechanic 
+
+Warrior should prob get like 5 extra armour and not get the health nerf from !2501
```

Comment:
Warrior is already strong. Increasing armor by five would put it similar to prae's and base ravager which i am opposed towards as it is a T2.